lib/standalone.js

```diff
@@ -138,7 +138,7 @@ async function standalone({
 		const autofix = fix ?? config?.fix ?? false;
 
 		if (autofix && postcssResult && !postcssResult.stylelint.ignored) {
-			returnValue.output =
+			returnValue.code =
```

Comment:
minor

The newly added LinterResult.code should be `string | undefined` but `!postcssResult.stylelint.disableWritingFix` is boolean.

Reply:
Um, if `!postcssResult.stylelint.disableWritingFix === false`, then run `returnValue.code = code`, right?

Reply:
foo = false && whatever

foo will always be false

Reply:
My bad, prettier tricked me.